init.lua

```diff
@@ -232,6 +236,8 @@ local function windowEventHandler(window, event, self)
     after the window was added ]]
     --
 
+    if is_floating[window:id()] then return end
```

Comment:
You might want to see if you could use the `windowDestroyed` event to remove a window from the `is_floating` list when the user closes the window. We don't use `windowDestroyed` for tiling because MacOS sends a `windowNotVisable` event before `windowDestroyed` and this is more generic across minimized, hidden, and closed window actions. But I think if a user minimizes or hides a floating window, it should come back as floating when restored.